engine/server.ts

```diff
@@ -185,6 +200,25 @@ export class ChannelEngine {
     if (options && options.useDemuxedAudio) {
       this.useDemuxedAudio = true;
     }
+    this.dummySubtitleEndpoint = dummySubtitleEndpointPath;
```

Comment:
I am not really sure I understand what you try to accomplish here below. Don't understand the logic here, can this be simplified? Rewrite this to not overwrite input parameters

Reply:
fixed